debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoUtil.java

```diff
@@ -219,6 +220,21 @@ public static ServerAddress parseAddress(String addressStr) {
         return null;
     }
 
+    /**
+     * Helper function to extract the session transaction-id from an oplog event.
```

Comment:
Where is the "session" term coming from? Isn't this id unique on a global level?

Reply:
The "session" term is what Mongo uses to refer to `lsid`, aka Logical Session ID.

This value is the same across all transactions and operations.  This is why the combination of both the `lsid` and the `txnNumber` are important because the `txnNumber` field merely starts at 1 and increments for every operation/transaction that happens.  

If the client disconnects & re-connects, it gets a new `lsid` and `txnNumber` will start from 1 again and continue to increase per transaction scoped operation.

Reply:
What/who is the "client" in this context? It's not Debezium, right? I.e. if we crash and don't commit offsets and re-read specific log sections a second time, we'd see the same lsid and txnNumber as before, correct?